### DISCUSSION TITLE
3D objects rendered as 2D in AR on Android

### DISCUSSION THREAD

**Discussion by Bketting** [google/model-viewer#2550] [Q&A]
Starting today, our model-viewer implementation renders 3D models als flat 2D objects in Augmented Reality on Android devices. This wasn’t a problem before, and nothing changed on our end. Anyone else come across this problem?

Example: https://floatscans.com/models/aronson

**Comment by elalish** [comment]
That sounds very strange; can you include a screen recording? It seems to work as expected on my Pixel 3. Can you be specific about the device/OS/browser?
